Sage-Bionetworks/sage-monorepo
# Added storage manifest endpoint for schematic

## Changelog
Goal: explore adding an endpoint in mono repo
* Added `storage/asset-views/{assetView}/projects/{projectId}/manifests` endpoint
* Added authentication token header for authentication 
* Added schematicpy23.1.1 in api project

##To test this PR
* nx serve schematic-api
* Copy and pasted synapse access token to authenticate
* Use the following asset view: 
- asset view: syn23643253
- project id: syn30988314
See response 200
